src/styles/blocks/card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  transition: transform $global-transition-time ease;
+
+  &__picture {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 3px;
+    margin-bottom: 40px;
```

Comment:
The margin-bottom property here is too large. It is causing too much space between the image and the title. Consider reducing it.